deploy-as-code/helm/charts/digit-works/backend/works-inbox-service/values.yaml

```diff
@@ -72,3 +72,13 @@ env: |
       configMapKeyRef:
         name: egov-config
         key: egov-state-level-tenant-id
+  - name: EGOV_USER_HOST
+    valueFrom:
+      configMapKeyRef:
+        name: egov-service-host
+        key: egov-user
+  - name: EGOV_DASHBOARD_ANALYTICS_HOST
```

Comment:
As we haven't deployed this, don't write it here. I don't think it is a hard dependency for inbox.

Reply:
removed.